packages/core/js-client/package.json

```
@@ -0,0 +1,73 @@
+{
+    "name": "@fluencelabs/js-client",
+    "version": "0.0.10",
+    "description": "TypeScript implementation of Fluence Peer",
```

Don't know what would be the good description. But would be good to think about something appropriate
```suggestion
    "description": "js-client for Fluence",
```

---

Changed it. You can check again